Mathlib/RingTheory/MvPowerSeries/Evaluation.lean

```diff
@@ -60,6 +60,7 @@ variable {φ : R →+* S}
 open WithPiTopology
 
 /-- Families at which power series can be consistently evaluated -/
+@[mk_iff hasEval_def]
 structure HasEval (a : σ → S) : Prop where
   hpow : ∀ s, IsTopologicallyNilpotent (a s)
```

Comment:
is it worth the change?
```suggestion
  isTopologicallyNilpotent s := IsTopologicallyNilpotent (a s)
```

Reply:
(There will be some consequences…)

Reply:
I think I prefer to open another PR just for this, even though the scope of the renaming is quite limited.

Reply:
OK. Then, since I'm responsible for the bad naming, I'll do that PR.